modules/mono/editor/GodotTools/GodotTools/Utils/OS.cs

```diff
@@ -37,7 +37,7 @@ public static class Names
         public static class Platforms
         {
             public const string Windows = "windows";
-            public const string OSX = "osx";
+            public const string macOS = "macos";
```

Comment:
This string literal should not be changed. `OS.Platforms` provides the "platform" names, a.k.a. the name expected by SCons when compiling, the name of the folders in the [platform directory](https://github.com/godotengine/godot/tree/182d6d6f4273f79a7b379f69d33249df5b74e670/platform), which is still `osx` for macOS.

Reply:
For this PR yes, but note that this will likely change for 4.0: https://github.com/godotengine/godot-proposals/issues/1161

Reply:
That proposal is not approved yet. For now we follow upstream, the SDK is called MacOSX.